client/src/widget/EmbeddedScheduler.js

```diff
@@ -0,0 +1,184 @@
+// Esssentially a copy of the `Scheduler.js` component
```

Comment:
Is there a way to just use the Scheduler component (maybe with some extra props) to avoid repeating code?